Ginger/GingerCoreNETUnitTest/TestResources/CLI/CLI-NotExisting_OnlyNames.Ginger.AutoRunConfigs.json

```diff
@@ -0,0 +1,85 @@
+{
+  "SolutionLocalPath": "SOLUTION_PATH",
+  "ShowAutoRunWindow": false,
+  "VerboseLevel": "normal",
+  "Runset": {
+    "Exist": false,
+    "Name": "Calc_Test_Dynamic",
+    "EnvironmentName": "Env1",
+    "RunAnalyzer": false,
+    "RunInParallel": true,
+    "Runners": [
+      {
+        "Name": "Runner Dynamic",
+        "EnvironmentName": "Env2",
+        "OnFailureRunOption": "ContinueToRunall",
+        "AppAgentMappings": [
+          {
+            "ApplicationName": "app1",
+            "AgentName": "IE",
+          }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected character ('}' (code 125)): was expecting double-quote to start field name](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4969846)